test/webserv/integration/test_http_get.py

```
@@ -49,6 +23,27 @@ def send_request_and_assert_response(request_file, expected_response):
             REQUEST_GET_2XX_DIR + "200_03_sub_connection_close.txt",
             response_header_get_sub_200_close + sub_index_file,
         ),
+        (
+            REQUEST_GET_2XX_DIR
+            + "200_04_connection_keep_alive_and_200_connection_keep_alive.txt",
+            response_header_get_root_200_keep
```

Ruff

`response_header_get_root_200_keep` may be undefined, or defined from star imports

(F405)